tests/Http/Three/SignupTest.php

```diff
@@ -268,19 +269,24 @@ public function testValidationForUpdatingASignup()
     }
 
     /**
-     * Test that non-authenticated user's/apps can't update signups.
+     * TTest that a non-admin or user that doesn't own the signup can't update signup.
```

Comment:
"TTest" 😬

Reply:
yikes - thanks for catching that! not supposed to be a stutter :)